Project.toml

```diff
@@ -5,16 +5,20 @@ version = "0.2.1"
 
```

Comment:
Could you please bump the minor version here? (Sorry, missed that before)

Reply:
Done.